op-simulator/op_simulator.go

```diff
@@ -22,7 +22,8 @@ const (
 )
 
 type Config struct {
-	Port uint64
+	Port        uint64
+	BaseChainId uint64
```

Comment:
`SourceChainId` is also an option for this it's what we call it in viem if you wanted to follow a similar pattern. Another thing when it comes to naming variables/functions with `Id` in it you'll normally see it written as `BaseChainID`